compensated-ruby/spec/gumroad/event_parser_spec.rb

```diff
@@ -64,6 +64,25 @@ def fixture_data(fixture)
         end
       end
 
+      describe "#pretransformed_data(body_or_data)" do
+        subject(:data) { event_parser.pretransformed_data(body_or_data) }
+        let(:gumroad_event_keys) { ["seller_id", "product_id", "product_name", "permalink", "product_permalink", "email", "price", "currency", "quantity", "order_number", "sale_id", "sale_timestamp", "purchaser_id", "subscription_id", "is_recurring_charge", "recurrence", "is_gift_receiver_purchase", "refunded", "resource_name", "full_name", "disputed", "dispute_won", "retry_count"] }
```

Comment:
I noticed the key for gumroad is going to be string. To be consistent with other `#pretransformed_data`, I was thinking to pull in `deep_symbolize_keys` but then it seems too much to pull in active support.
Maybe tell me this isn't important?

Reply:
I think that's a good point that we want to be consistent; let's go with symbolizing the keys but try to avoid pulling in a dependency

Reply:
I'm now worried that if we do this, we are going to break functionality for people who built onto the gumroad adapter :(.